frontend/src/pages/class/Insight.jsx

```diff
@@ -34,12 +33,31 @@ const columns = [
   {
     field: 'correctness',
     headerName: 'Gem. correctheid',
-    flex: 0.35,
+    flex: 0.5,
     valueGetter: (params) => params.row.correctness,
   },
 ];
 
-const Insight = () => {
+const getRows = (studentResults) => {
+  const distribution = studentResults[0];
+  const studentInfo = studentResults[1];
+  const rows = [];
+  if (studentInfo && distribution) {
+    for (let i = 0; i < distribution.length; i++) {
```

Comment:
Ik zie op dit moment nog veel geïndexte forloops in de code. Dat is niet persé fout, maar het zou bijvoorbeeld ook genoteerd kunnen worden als:

`for (const ... of distribution) {
   ...
}`

Op deze manier heb je dan geen `[i]` nodig iedere keer in je code, maar kan je refereren naar de const in je forloop

Reply:
wederom luiheid om niet op te zoeken hoe (for .. in distribution) in python om te zetten naar js  🥴